compiler/src/iree/compiler/Dialect/HAL/IR/HALDialect.cpp

```diff
@@ -115,6 +117,30 @@ class HALToVMConversionInterface : public VMConversionDialectInterface {
   }
 };
 
+class HALAffinityAnalysisDialectInterface
+    : public IREE::Stream::AffinityAnalysisDialectInterface {
+public:
+  using AffinityAnalysisDialectInterface::AffinityAnalysisDialectInterface;
+  IREE::Stream::LayoutAttrSolverFn
+  makeLayoutAttrSolver(ModuleOp moduleOp) const {
+    return [=](IREE::Stream::AffinityAttr aff, Operation *op,
```

Comment:
always use full names for things wherever possible - it's useful hints to readers as to what the type is and how it is treated - shortening to arbitrary forms makes it impossible to find/replace in codebases and more difficult for non-native speakers (here and anywhere else you've shortened things).

```suggestion
    return [=](IREE::Stream::AffinityAttr affinityAttr, Operation *op,
```

Reply:
I see, done.